components/trustpilot/actions/fetch-product-reviews/fetch-product-reviews.mjs

```diff
@@ -3,8 +3,8 @@ import trustpilot from "../../trustpilot.app.mjs";
 export default {
   key: "trustpilot-fetch-product-reviews",
   name: "Fetch Product Reviews",
-  description: "Retrieves a list of product reviews for a specific business unit on Trustpilot. This action enables you to fetch multiple product reviews with powerful filtering options including star ratings, language, tags, and sorting preferences. Ideal for monitoring product feedback trends, generating reports, analyzing customer sentiment across your product catalog, or building review dashboards. Supports pagination for handling large review volumes. [See the documentation](https://developers.trustpilot.com/product-reviews-api#get-private-product-reviews)",
-  version: "0.0.3",
+  description: "Retrieves a list of product reviews for a specific business unit.",
+  version: "0.1.0",
```

Comment:
Since the prop names are being changed, we need to bump the major version (e.g. to `1.0.0`), as users who update the component in their workflows will lose their existing configuration